.github/dependabot.yml

```diff
@@ -9,7 +9,7 @@ updates:
     allow:
       # Allow both direct and indirect updates for all packages
       - dependency-type: all
-    open-pull-requests-limit: 0
+    # open-pull-requests-limit: 0
```

Comment:
How is this related to security updates?

And in general please remove the line if you intend to remove it.

Reply:
open-pull-requests-limit limits the numer of dependabot pull requests for general updates.
It does not apply to security updates, so regardless of the limit dependabot will create security PRs.
Setting the limit to 0 is an undocumented workaround, because there is no other way to tell dependabot to only do securoty updates.

Reply:
removed the line